cibyl/models/attribute.py

```diff
@@ -15,12 +15,49 @@
 """
 from dataclasses import dataclass
 
+from cibyl.cli.argument import Argument
+
 
 @dataclass
 class AttributeValue():
     """Represents the value used by the attributes of the different models"""
 
     name: str
-    type: object
+    attr_type: object
     value: object
-    arguments: list
+    arguments: list[Argument]
+
+
+class AttributeListValue(AttributeValue):
+    """Represents a list of AttributeValue objects"""
+
+    def __init__(self, name, arguments=None, attr_type=None, value=None):
+
+        super(AttributeListValue, self).__init__(
```

Comment:
Here we still need to pass value to the init method of AttributeValue I think

Reply:
+1
The value is lost if it is neither a list nor 'None'.

Reply:
We do, on line 37. Unless I misunderstood you.

Reply:
On the call to the super's init I don't see the value parameter, maybe it was lost when pushing? I only see name, arguments and attr_type passed

Reply:
oh k got it now. thanks, will fix it